src/core/thread/mlr_manager.hpp

```diff
@@ -142,6 +142,8 @@ class MlrManager : public InstanceLocator, private NonCopyable
 #endif
 
 private:
+    static constexpr uint32_t kUpdateLocalSubscriptionsDelay = 10;
```

Comment:
```suggestion
    static constexpr uint32_t kUpdateLocalSubscriptionsDelay = 100;
```
10ms may not be enough considering possible OS scheduling delay and THCI script delays.